coffee/visitors/utilities.py

```diff
@@ -284,7 +283,9 @@ def default_retval(cls):
     """
 
     def __init__(self, symbols):
+        from coffee.utils import flatten
         self.symbols = symbols
+        self.flatten = flatten
```

Comment:
work around to circular imports, as I'm importing ``utils`` and ``utils`` is importing some visitors. Not sure where I should move flatten ? the other option is to re-import flatten in the member methods that use it